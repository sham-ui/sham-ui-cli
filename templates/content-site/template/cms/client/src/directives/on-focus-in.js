import { EventListener } from 'sham-ui-directives';

export class onFocusIn extends EventListener {
    constructor() {
        super( 'focusin' );
    }
}
