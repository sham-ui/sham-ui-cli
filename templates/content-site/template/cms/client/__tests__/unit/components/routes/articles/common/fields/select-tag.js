import * as directives from 'sham-ui-directives';
import { onFocusIn } from '../../../../../../../src/directives/on-focus-in';
// eslint-disable-next-line max-len
import RoutesArticlesCommonFieldsSelectTag  from '../../../../../../../src/components/routes/articles/common/fields/select-tag.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RoutesArticlesCommonFieldsSelectTag, {}, {
        directives: {
            onFocusIn,
            ...directives
        }
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
