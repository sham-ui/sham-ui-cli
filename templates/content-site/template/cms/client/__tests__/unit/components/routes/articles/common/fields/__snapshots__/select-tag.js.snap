// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "onChange": [Function],
    "query": "",
    "showItems": false,
  },
  "html": "
<div class=\\"autocomplete-input\\">
  <input autocomplete=\\"off\\">
  <!--0-->
</div>
",
}
`;
